horace_core/sqw/page_operations/PageOpBase.m

```diff
@@ -210,10 +218,10 @@
             % out_obj -- sqw object created as the result of the operation
             % obj     -- nullified PageOp object.
 
-            pix = obj.pix_;
-            pix     = pix.set_data_range(obj.pix_data_range_);
-            % as we normally read data and immediately dump them back, what
-            % is the point of converting them to double and back?
+            pix   = obj.pix_;
+            pix   = pix.set_data_range(obj.pix_data_range_);
+            % revert usual order of pixel operations (data converted to
+            % double)
```

Comment:
How does this explain conversion to double? "Revert usual order of ... operations" usually implies instead of doing `(a * b) + c` we do `a * (b + c)` to me.